Update docs/quickstart-guide.md (#41)
st-tech/gatling-operator: docs/quickstart-guide.md

rolebinding.rbac.authorization.k8s.io/gatling-operator-leader-election-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/gatling-operator-manager-rolebinding created
deployment.apps/gatling-operator-controller-manager created
```

The CRD, Manager, and other resources are deployed and ready to run Gatling CR.

The example uses a manifest of v0.5.0. Please change the version if necessary. You can check the version from the [Release page](https://github.com/st-tech/gatling-operator/releases).

## Deploy Gatling CR
Deploy the Gatling CR using the [example](https://github.com/st-tech/gatling-operator/tree/main/config/samples) in the gatling-operator repository.
```
kustomize build config/samples | kubectl apply -f -
```

The output is similar to:

```bash
serviceaccount/gatling-operator-worker unchanged
role.rbac.authorization.k8s.io/pod-reader unchanged
rolebinding.rbac.authorization.k8s.io/read-pods configured
secret/gatling-notification-slack-secrets unchanged
gatling.gatling-operator.tech.zozo.com/gatling-sample01 created
```

After execution, the ServiceAccount and Gatling CR required to run the Gatling Runner Pod will be deployed.

> After deploying the Gatling CR, the Gatling CR, Gatling Runner Job, and Gatling Runner Pod will be generated and the Gatling test scenario will be executed.

Reviewer: How about this?
```suggestion
After deploying the Gatling CR, Gatling Controller creates a Gatling Runner job. The job then runs the Gatling Runner pods to execute Gatling load test scenario in parallel.
```

```
$ kubectl get gatling,job,pod
```

> The output is similar to:

Reviewer: ```suggestion
Expected output would be like this:
```

```
NAME                                                      AGE
gatling.gatling-operator.tech.zozo.com/gatling-sample01   10s

NAME                                COMPLETIONS   DURATION   AGE
job.batch/gatling-sample01-runner   0/3           9s         9s

NAME                                READY   STATUS    RESTARTS   AGE
pod/gatling-sample01-runner-8rhl4   1/1     Running   0          9s
pod/gatling-sample01-runner-cg8rt   1/1     Running   0          9s
pod/gatling-sample01-runner-tkplh   1/1     Running   0          9s
```

You can also see from the Pod logs that Gatling is running.

```
kubectl logs gatling-sample01-runner-tkplh -c gatling-runner -f
```

> The output is similar to:

Reviewer: ```suggestion
Expected output would be like this:
```

================================================================================
2022-02-25 06:08:40                                          14s elapsed
---- Requests ------------------------------------------------------------------
> Global                                                   (OK=6      KO=0     )
> request_1                                                (OK=1      KO=0     )
> request_1 Redirect 1                                     (OK=1      KO=0     )
> request_2                                                (OK=1      KO=0     )
> request_3                                                (OK=1      KO=0     )
> request_4                                                (OK=1      KO=0     )
> request_4 Redirect 1                                     (OK=1      KO=0     )

---- Scenario Name -------------------------------------------------------------
[##########################################################################]100%
          waiting: 0      / active: 0      / done: 1
================================================================================

Simulation MyBasicSimulation completed in 14 seconds
```

> In this example, the notification of Gatling result reports and the storage of the result reports in the cloud provider are not performed.

Reviewer: Here is my suggestion for this
```suggestion
As configured in [the sample manifest](https://github.com/st-tech/gatling-operator/blob/85e69840274214c47e63f65a5c807dd541dff245/config/samples/gatling-operator_v1alpha1_gatling01.yaml#L6-L8), an aggregated Gatling HTML report is not created, nor notification result message is posted.
```

> This can be done by setting the .spec.cloudStorageSpec and .spec.notificationServiceSpec.

Reviewer: Here is my suggestion for this
```suggestion
You can generate the Gatling HTML report by enabling `.spec.generateReport` flag and setting the `.spec.cloudStorageSpec`. Also you can posting the notification message by enabling `.spec. notifyReport ` and setting `.spec.notificationServiceSpec`. For more information about configuring Gatling CR, please refer to [User Guide](./user-guide.md)
```
